general/forall.hpp

```diff
@@ -87,58 +87,94 @@ void OmpWrap(const int N, HBODY &&h_body)
 
 
 /// RAJA Cuda backend
-#if defined(MFEM_USE_RAJA) && defined(RAJA_ENABLE_CUDA)
+#if defined(MFEM_USE_RAJA) && defined(RAJA_DEVICE_ACTIVE)
+
+#if defined(RAJA_CUDA_ACTIVE)
+using launch_policy =
+   RAJA::expt::LaunchPolicy<RAJA::expt::null_launch_t, RAJA::expt::cuda_launch_t<false>>;
+using teams_x =
+   RAJA::expt::LoopPolicy<RAJA::loop_exec,RAJA::cuda_block_x_direct>;
+using threads_z =
+   RAJA::expt::LoopPolicy<RAJA::loop_exec,RAJA::cuda_thread_z_direct>;
+#else
```

Comment:
`#else if defined(RAJA_HIP_ACTIVE)` just to be safe?

Reply:
Don't have that one, but we refactored to check for RAJA_ENABLE_HIP.